frontend/pages/passport-application/index.tsx

```diff
@@ -0,0 +1,99 @@
+import Link from "next/link"
+
+import Logo from "../../shared-components/logo"
+import { USStateSelect } from "../../shared-components/state-select/UsStateSelect"
+import styles from "./passport.module.css"
+
+function handleSubmit() {
+  // TODO: add form submission logic
+}
+
+export default function Passport({ name = "Herbert Q. Placeholder" }) {
+  return (
+    <div className={styles.wrapper}>
```

Comment:
If you like, on other components we've been destructuring the classNames from the `styles` object (especially when there's more than like 1-3 classNames) 

It helps DRY up the template code a bit not having to write `styles.` repeatedly!